src/js/rest/getCustomize.js

```diff
@@ -0,0 +1,26 @@
+import createError from './common/createError';
```

Comment:
Because customize is a verb, so the file name should be changed
getCustomize => getCustomization